setup.py

```diff
@@ -69,4 +69,12 @@ def get_version(rel_path):
         packages=find_packages(exclude=["cloudevents.tests"]),
         version=pypi_config["version_target"],
         install_requires=["deprecation>=2.0,<3.0"],
+        extras_require={
+            "pydantic": [
+                # 1.9.0 does not support python 3.6.
+                # When issue https://github.com/cloudevents/sdk-python/issues/177
+                # is resolved we SHOULD update this dependency.
+                "pydantic>=1.0.0<1.9.0",
+            ],
```

Comment:
```suggestion
            "pydantic": [
                "pydantic>=1.0.0<1.9.0; python_version <= '3.6'",
                "pydantic>=1.0.0<2.0; python_version > '3.6'"
            ],
```
Can we try this approach?